src/Settings.ts

```diff
@@ -1,11 +1,13 @@
 export interface Settings {
     globalFilter: string;
     removeGlobalFilter: boolean;
+    reminder: boolean; //To activate the reminder input
```

Comment:
I will probably rename this to something that removes the need for a comment, perhaps `remindersEnabled` or `remindersActivated`